internal/core/api.rs

```diff
@@ -264,25 +405,23 @@ pub use crate::input::PointerEventButton;
 #[non_exhaustive]
 pub enum WindowEvent {
     /// The mouse or finger was pressed
-    PointerPressed { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    PointerPressed { position: LogicalPosition, button: PointerEventButton },
     /// The mouse or finger was released
-    PointerReleased { position: euclid::Point2D<f32, LogicalPx>, button: PointerEventButton },
+    PointerReleased { position: LogicalPosition, button: PointerEventButton },
     /// The position of the pointer has changed
-    PointerMoved { position: euclid::Point2D<f32, LogicalPx> },
+    PointerMoved { position: LogicalPosition },
     /// Wheel was rotated.
     /// `pos` is the position of the mouse when the event happens.
-    /// `delta` is the amount of pixel to scroll.
-    PointerScrolled {
-        position: euclid::Point2D<f32, LogicalPx>,
-        delta: euclid::Vector2D<f32, LogicalPx>,
-    },
+    /// `delta_x` is the amount of pixels to scroll in horizontal direction,
+    /// `delta_y` is the amount of pixels to scroll in vertical direction.
+    PointerScrolled { position: LogicalPosition, delta_x: f32, delta_y: f32 },
```

Comment:
how about `horizontal_delta` and `vertical_delta` ?  or do you think it is too verbose?

Reply:
I was wondering the same. What makes me think that x/y is better is that it combines better with a z in the future, as opposed to a … depth?

Reply:
The other source I was looking at was the DOM wheel event, which uses `deltaX`, `deltaY` and `deltaZ`: https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event